requirements.txt

```diff
@@ -24,3 +24,4 @@ networkx==2.1
 requests==2.20.0
 
 jsonschema==2.6.0
+marshmallow-dataclass==0.5.5
```

Comment:
`marshmaollow` itself should be added as well.

Reply:
It's already in raiden, so I add it only with a major version number?

Reply:
Pip is unable to properly resolve dependency versions. It will install a slightly more recent version than requested by raiden. The code still works with the exception of the `pkg_resources.require('raiden-services')[0].version` call which raises
```
pkg_resources.ContextualVersionConflict: (marshmallow 2.19.1 (/home/karl/tools/venv-services/lib/python3.7/site-packages), Requirement.parse('marshmallow==2.15.4'), {'raiden'})
```
It also causes a warning during install.

Do you know a clean solution to this?